src/gui/optionsdialog.ui

```diff
@@ -1457,6 +1457,9 @@ Manual: Various torrent properties (e.g. save path) must be assigned manually</s
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinPort">
+                   <property name="toolTip">
+                    <string>Set to 0 to let OS pick an unused port</string>
```

Comment:
... the OS...

Maybe translators need a hint on what OS means? Unless you believe it's a well known abbreviation.

Reply:
There is no `0` value displayed. Shouldn't you mention `Any` instead?

Reply:
Is it possible to set the port to "Any" without the user knowing it's 0? I think 0 should be mentioned anyway but i suppose the display value could also be mentioned: `0 ("Any")`. I might be confused what "Any" means without the quotes if i didn't know it represents the 0.

Reply:
>Maybe translators need a hint on what OS means? 

Or how about `Set to 0 to let your system pick an unused port`?

>There is no 0 value displayed. Shouldn't you mention Any instead?

Currently the WebUI shows `0` instead of `Any`.
Another reason is, user can input `0` but cannot type `"Any"` into the spinbox.

>i suppose the display value could also be mentioned: 0 ("Any"). I might be confused what "Any" means without the quotes if i didn't know it represents the 0.

I had the same concern so I didn't wrote it out.

Reply:
>Or how about `Set to 0 to let your system pick an unused port`?

LGTM

Reply:
> > Or how about `Set to 0 to let your system pick an unused port`?
> 
> LGTM

Done.